README.md

```diff
@@ -45,86 +42,105 @@ Get the following:
    - [Linux](https://docs.docker.com/desktop/install/linux-install/)
 7. [NumPy](https://numpy.org/install/)
 
-> **TIP:** If you're a macOS user, you can also use `brew` to install NumPy:
+   > **TIP:** If you're a macOS user, you can also use `brew` to install NumPy:
 
```

Comment:
```suggestion
> 
```
To make the command below part of the TIP.

Reply:
Changed